daffodil-test/src/test/resources/META-INF/services/org.apache.daffodil.runtime1.layers.api.Layer

```diff
@@ -27,8 +27,31 @@
 # of the layering features of Daffodil. They can be reused to create "real"
 # layering, but one should expect to have to adapt their code substantially.
 #
-org.apache.daffodil.runtime1.layers.AISPayloadArmoringLayerCompiler
-org.apache.daffodil.runtime1.layers.CheckDigitLayerCompiler
-org.apache.daffodil.runtime1.layers.IPv4ChecksumLayerCompiler
+org.apache.daffodil.runtime1.layers.AISPayloadArmoringLayer
+org.apache.daffodil.runtime1.layers.CheckDigitLayer
+org.apache.daffodil.runtime1.layers.ipv4.IPv4ChecksumLayer
+org.apache.daffodil.runtime1.layers.AllTypesLayer
+org.apache.daffodil.runtime1.layers.STL_Ok1
+org.apache.daffodil.runtime1.layers.STL_Ok2
+org.apache.daffodil.runtime1.layers.STL_Ok3
+org.apache.daffodil.runtime1.layers.STL_Ok4
+org.apache.daffodil.runtime1.layers.STL_BadTypeInLayerCode1
+org.apache.daffodil.runtime1.layers.STL_BadTypeInLayerCode2
+org.apache.daffodil.runtime1.layers.STL_BadMissingSetter
+org.apache.daffodil.runtime1.layers.STL_BadMissingSetterArg
+org.apache.daffodil.runtime1.layers.STL_BadMissingGetter
+org.apache.daffodil.runtime1.layers.STL_BadMissingSetterVar
+org.apache.daffodil.runtime1.layers.STL_BadMissingGetterVar
+org.apache.daffodil.runtime1.layers.STL_BadMissingDefaultConstrutor
+org.apache.daffodil.runtime1.layers.STL_BadNotALayer # error intentional. This isn't a layer class
+org.apache.daffodil.runtime1.layers.STL_BombOutLayer
+
+
+
+
+
+
+
+
```

Comment:
Are the empty lines at the end intentional?

Reply:
Inadvertent. It's another IDE setting to always truncate final empty lines.